README.md

```diff
@@ -276,4 +308,123 @@ See [utils/usage.ipynb](./utils/usage.ipynb) for an example.
 
 ---
 
-**Note:** These AI Starter Kit code samples are provided "as-is," and are not production-ready or supported code. Bugfix/support will be on a best-effort basis only. Code may use third-party open-source software. You are responsible for performing due diligence per your organization policies for use in your applications.
+## Setting Up Your Environment
```

Comment:
this section should be before  ### 4. Run the desired starter kit

Reply:
This has now moved.